third_party/move/move-vm/runtime/src/interpreter.rs

```diff
@@ -980,6 +979,9 @@ where
                 }
             }
         }
+        self.ref_state.in_borrow_field_mut_api =
```

Comment:
See comment in runtime ref checks. This should not be required.

Reply:
removed